packages/did-core-test-server/suites/did-consumption/did-jsonld-consumption.js

```diff
@@ -0,0 +1,40 @@
+const deepEqual = require('deep-equal')
```

Comment:
Some of the tests using deepEqual, but there is a standard matcher checking for equality: `toBeEqual`.
Of course, the results are the same. Maybe we can refactor later.

FYI: [What is the difference between 'toBe' and 'toEqual' in Jest?](https://stackoverflow.com/questions/45195025/what-is-the-difference-between-tobe-and-toequal-in-jest)

Reply:
Yes, I will refactor this in a future PR. We should be using the .toEqual() test in Jest. I didn't know it existed at the time I pulled in deep-equal.